aws_lambda_powertools/utilities/data_classes/code_pipeline_job_event.py

```diff
@@ -80,6 +80,8 @@ def location(self) -> CodePipelineLocation:
 
 
 class CodePipelineArtifactCredentials(DictWrapper):
+    _str_blacklist = ["secret_access_key", "session_token"]
```

Comment:
Instead of “blacklist” can we call it “redaction_list” or something similar? To include the property but mask the values.

Reply:
Thank you. I've renamed it to `_str_sensitive_data` which is probably clearer too.